test/persistence-hooks.suite.js

```diff
@@ -1202,9 +1227,14 @@ module.exports = function(dataSource, should) {
       });
     });
 
-    function pushContextAndNext() {
+    function pushContextAndNext(fn) {
       return function(context, next) {
+        if (typeof fn === 'function') {
+          fn(context);
+        }
+
         context = deepCloneToObject(context);
+        context.hookState.test = true;
```

Comment:
This is just a variable that will be set by all invocations of `pushContextAndNext` and thus, it will make sure that all tests will verify that the hookState is working (and available to all known hooks that invoke `pushContextAndNext`, which should be all IIRC).